aws/cloud-init/master.script.tpl

```diff
@@ -97,11 +97,6 @@ if [ "$${INSTALL_LEADER}" = "$${EC2_INSTANCE_ID}" ] && [ ! -f /tmp/gravity ]; th
     /usr/local/bin/tele pull -o /tmp/installer.tar ${source}
   fi
 
-  ADVERTISE=""
-  if [ ! -z "${ops_advertise_addr}" ]; then
-    ADVERTISE="--ops-advertise-addr=${ops_advertise_addr}"
```

Comment:
Hm, so with this change it is no longer possible to provision a Hub using this terraform. I would keep it backwards compatible and instead of changing the meaning of the `ops_advertise_addr` variable introduced a new one for explicitly requesting a gravity-public service for a regular non-Hub cluster.